egs/swbd/s5c/local/run_asr_segmentation.sh

```diff
@@ -0,0 +1,145 @@
+#! /bin/bash
```

Comment:
I did not run this. @ngoel17 has the WER results. He can create a PR based on how he used this.